apps/wallet/src/ui/app/pages/home/transfer-coin/index.tsx

```diff
@@ -1,238 +1,135 @@
+        () => Coin.computeGasBudgetForPay(allCoinsOfTransferType, 5000n),
+        [allCoinsOfTransferType]
     );
-    const { gasBudget: gasBudgetEstimation, isLoading } = useGasBudgetInMist(
+    const { gasBudget: gasBudgetEstimation } = useGasBudgetInMist(
         gasBudgetEstimationUnits
     );
-    const validationSchemaStepOne = useMemo(
-        () =>
-            createValidationSchemaStepOne(
-                coinType || '',
-                coinBalance,
-                coinSymbol,
-                gasAggregateBalance,
-                coinDecimals,
-                gasDecimals,
-                gasBudgetEstimation || 0,
-                maxSuiSingleCoinBalance
-            ),
-        [
-            coinType,
-            coinBalance,
-            coinSymbol,
-            coinDecimals,
-            gasDecimals,
-            gasAggregateBalance,
-            gasBudgetEstimation,
-            maxSuiSingleCoinBalance,
-        ]
-    );
-    const validationSchemaStepTwo = useMemo(
-        () => createValidationSchemaStepTwo(),
-        []
-    );
 
-    const dispatch = useAppDispatch();
     const navigate = useNavigate();
-    const onHandleSubmit = useCallback(
-        async (
-            { to, amount }: FormValues,
-            { resetForm }: FormikHelpers<FormValues>
-        ) => {
-            if (coinType === null || !gasBudgetEstimationUnits) {
-                return;
-            }
-            setSendError(null);
-            trackEvent('TransferCoins', {
-                props: { coinType },
-            });
-            try {
-                const bigIntAmount = parseAmount(amount, coinDecimals);
-                const response = await dispatch(
-                    sendTokens({
-                        amount: bigIntAmount,
-                        recipientAddress: to,
-                        tokenTypeArg: coinType,
-                        gasBudget: gasBudgetEstimationUnits,
-                    })
-                ).unwrap();
-
-                resetForm();
-                const txDigest = getTransactionDigest(response);
-                const receiptUrl = `/receipt?txdigest=${encodeURIComponent(
-                    txDigest
-                )}&from=transactions`;
-
-                navigate(receiptUrl);
-            } catch (e) {
-                setSendError((e as SerializedError).message || null);
-            }
-        },
-        [dispatch, navigate, coinType, coinDecimals, gasBudgetEstimationUnits]
-    );
 
-    const handleNextStep = useCallback(
-        (_: FormValues, { setSubmitting }: FormikHelpers<FormValues>) => {
-            setCurrentStep((prev) => prev + 1);
-            setSubmitting(false);
-        },
-        []
-    );
+    const closeSendToken = useCallback(() => {
+        navigate('/');
+    }, [navigate]);
 
-    const handleBackStep = useCallback(() => {
-        setCurrentStep(DEFAULT_FORM_STEP);
-    }, []);
+    const onHandleSubmit = useCallback(() => {
```

Comment:
let move that to the button - onClick attribute

Reply:
see the other PR https://github.com/MystenLabs/sui/blob/02e15562701aa7369d99062c112bd423c2535820/apps/wallet/src/ui/app/pages/home/transfer-coin/index.tsx#L175